src/mapml-viewer.js

```diff
@@ -354,6 +354,18 @@ export class MapViewer extends HTMLElement {
       totalSize += 49;
       this._fullScreenControl = M.fullscreenButton().addTo(this._map);
     }
+    if (!this._geolocationButton){
+      this._geolocationButton = M.geolocationButton({
+        showPopup: false,
+        strings: {
+          title: "Locate"
+        },
+        position: "bottomright",
+        locateOptions: {
+          maxZoom: 16
+        },
+      },this._map);
```

Comment:
I believe this should be:

```javascript
  this._geolocationButton = M.geolocationButton({...}).addTo(this._map)
```
but it seems to work??

Reply:
I made geolocationButton a function in the geolocationButton.js file